tests/unittests/core/io/conftest.py

```diff
@@ -59,10 +59,10 @@ def child_config(parent_config):
 
 
 @pytest.fixture
-def experiment_name_conflict(create_db_instance, parent_config, child_config):
+def experiment_name_conflict(pdatabase, parent_config, child_config):
     """Generate an experiment name conflict"""
-    create_db_instance.remove("experiments", {"name": "test"})
-    create_db_instance.remove("experiments", {"name": "test2"})
-    create_db_instance.write("experiments", parent_config)
-    create_db_instance.write("experiments", child_config)
+    pdatabase.remove("experiments", {"name": "test"})
```

Comment:
If the id of `test` and `test2` are the same as for parent_config and child_config, then we could use storage.delete_experiment(uid) instead of pdatabase.

Reply:
Should maybe add a `where` argument to this one too eh :)

Reply:
For now I did a fetch/delete